WordPress/Classes/Utility/WPError.m

```diff
@@ -88,7 +89,7 @@ + (void)showNetworkingAlertWithError:(NSError *)error title:(NSString *)title
     }
 
     if (message == nil) {
-        message = [error localizedDescription];
+        message = [[error localizedDescription] stringByStrippingHTML];
```

Comment:
Would it be better to do this inside `showAlertWithTitle:message:withSupportButton:okPressedBlock:`?